gitutil.py

```diff
@@ -151,6 +158,11 @@ def _has_stash(self):
     def _pop_stash(self):
         self.repo.git.stash('pop', '--quiet')
 
+    def _config_value(self, section, option, level:ConfigLevel=None):
+        if level:
+            level = level.value
```

Comment:
not required - always use level.value

btw: why is this required? I know what the cfg levels mean on git-side. just wondering why we need to discriminate now.

Reply:
I dont think it is needed for the introduced changes, as the api mentions
> If None, all applicable levels will be used

I'd rather do it properly, however.